src/Averager.java

```diff
@@ -0,0 +1,26 @@
+public class Averager extends Roll{
```

Comment:
You shouldn't be extending `Roll` here (in fact, you don't need to extend anything in this whole assignment).  Extending is sort of like inheritance in JavaScript.  

Are you extending your classes because you want to be able to access the stuff in Roll from Averager?  If so, the proper way to do that is to follow the pattern you use in your test files:
```
assertArrayEquals(expected, Averager.intergerify(input));
```
In the snippet above, `Averager` is the file that contains `integerify()` - that's al you need to do to make sure the current file knows about the contents of `Averager`